README.rst

```diff
@@ -14,6 +14,21 @@ and latest releases of the standard.
 
 Documentation
 -------------
-
 For more information visit `the documentation page
 <http://emva1288.readthedocs.io/en/latest/>`__
+
+Camera Testing : Pytest
+-------------
+Install Requirements : *pip install -r requirements.txt*
+
+execute a subset of smoke tests for the camera:
+
+- **pytest -s -m smoke**
```

Comment:
This does not work unless the user creates logs folder inside tests folder, probably because the logger configuration is not correct.

Reply:
Ah yes good point,  It should explicitly be noted to create the folder in the readme documentation, (or add an option in the logger to create the folder, if it does not already exist). The logs should work correctly, once the folder exists